app/controllers/admin/inventory_items_controller.rb

```diff
@@ -14,14 +14,14 @@ class InventoryItemsController < ResourceController
 
     private
 
-    # Overriding Spree method to load data from params here so that
+    # Overriding resource_controller method to load data from params here so that
     # we can authorise #create using an object with required attributes
     def build_resource
-      if parent_data.present?
```

Comment:
why are we removing this? are we really sure?

Reply:
yes :+1: parent data is used when there is a parent like product on the images controller or the order in the adjustments controller...